### PR TITLE
Pass py_binary data deps through to underlying binary.

### PR DESCRIPTION
**Pull request by optimality**
Also cleaning up the style a bit.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by pshields** [comment]
I think this would fix an issue I ran into. It looks like without this change, the data deps to py_appengine_binary  get silently dropped.

**Comment by pshields** [comment]
Can someone take a look at this merge it in? It looks like a pretty trivial fix.

A workaround in the meantime is to create a separate py_library and put the data argument on that. Then include that py_library as a dep to py_appengine_binary.

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: appengine/py_appengine.bzl [hunk 1]
```diff
@@ -199,8 +199,9 @@ def py_appengine_binary(name, srcs, configs, deps=[], data=[]):
   # uses py_binary because it generates __init__.py files
   native.py_binary(
       name="_py_appengine_" + name,
-      srcs = srcs,
+      srcs=srcs,
```
Comment (pmbethe09) [inline appengine/py_appengine.bzl]: This is not a cleanup.  Skylark style has spaces around '=' which is different than python

**Comment by jo2y** [comment]
FWIW, I noticed the lack of 'data = data' while working on #71 and my PR includes the same fix.

**Pull request closed without merging**